docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3726 @@
+Frequently Asked Questions
+--------------------------
+
+Q: I used a playlist link with some of the playlist related commands and it tells me that it can't find it. 
+How can I use this playlist link with playlist commands in audio?
+
+	Red Audio uses Red playlists in its commands that take playlist arguments. 
+	These playlists can be created and modified using the ``[p]playlist`` group command.
+	When a playlist or song(s) are saved as a Red playlist, it is assigned an ID automatically,
+	and it is also assigned the one-word name you provided it when creating the playlist.
+	Either one of these identifiers can be used with playlist-related commands.
+
+Q: How do I turn off autoplay?
+
+	Use the ``[p]audioset autoplay toggle`` command.
+
+Q: How do I get the bot to disconnect from the channel when it's done playing?
+
+	``[p]audioset dc`` will make the bot auto-disconnect when playback completes and the 
+	queue is empty. 
+	``[p]audioset emptydisconnect`` with a seconds argument greater than 0 will make the bot 
+	auto-disconnect once it's alone in the channel, after the amount of seconds given to the 
+	command. This setting takes precedence over ``[p]audioset dc`` if both settings are active.
+
+Q: How do I use localtracks?
+
+	See the :ref:`local tracks section<local-tracks>`.
+    
+Q: My console is saying that "Port 2333 is already in use". How can I fix this?
+
+    If you are trying to run multiple bots with Audio, you should follow our guide on
+    :ref:`setting up Audio for multiple bots<multibots>`. Otherwise, another external process
+    is using the port, so you need to figure out what is using port 2333 and terminate/disconnect it yourself.
+    
+Q: My terminal is saying that I "must install Java 11 for Lavalink to run". How can I fix this?
+
+    You are getting this error because you have a different version of Java installed, or you don't have
+    Java installed at all. As the error states, Java 11 is required, and can be installed from
+    `here <https://adoptium.net/temurin/releases/?version=11>`_.
+    
+    Use ``[p]llset java <path_to_java_11_executable>``, to make Audio launch Lavalink with a
+    specific Java binary. To do this, you will need to locate your ``java.exe``/``java`` file
+    in your **Java 11 install**.
+    
+    Alternatively, update your PATH settings so that Java 11 is the one used by ``java``. However,
+    you should confirm that nothing other than Red is running on the machine that requires Java.
+
+.. _queue_commands:
+
+----------------------
+Queue Related Commands
+----------------------
+
+* ``[p]queue`` - Shows the queue of playing songs and current settings for the server for shuffle and repeat.
+* ``[p]remove`` - Remove a song from the queue. This command uses the track position in the queue for identification, e.g. ``[p]remove 10`` will remove
+  the 10th song in the queue.
+* ``[p]shuffle`` - Toggle random song playback from the queue, and shuffles the queue.
```

Comment:
[p]shuffle shouldn't shuffle the queue since it's just a toggle, [p]queue shuffle is the on demand shuffle